screens/Home/MyVcs/GetIdInputModal.tsx

```diff
@@ -87,6 +90,18 @@ export const GetIdInputModal: React.FC<GetIdInputModalProps> = props => {
                         color={Theme.Colors.tooltipIcon}
                       />
                     }
+                    triggerComponentStyles={{width: 32}}
+                    toolTipContent={
```

Comment:
In my view it will be good to keep it like this because previously we have only one title and description so keeping tool tip component inside custom component and passing details from calling component is fine but as part of the detailed view status tool tip we have to give information about both **verified** and **pending** status and here we can follow the button approach you have mentioned but if the list of the attributes that we need to pass grows and if we want to give different styles to the different data(customization) then every time we have to make lot of changes in the custom tool tip component. So I feel keeping the code this way gives more control and it is more maintainable and customizable.

Passing more properties from the calling components gives clear focus on what details we are expecting but it will make customization hard.